medusa/tv/series.py

```diff
@@ -1612,6 +1621,47 @@ def load_imdb_info(self):
         log.debug(u'{id}: Obtained info from IMDb: {imdb_info}',
                   {'id': self.series_id, 'imdb_info': self.imdb_info})
 
+    def prev_episode(self):
+        """Return the last aired episode air date.
+
+        :return:
+        :rtype: datetime.date
+        """
+        log.debug(u'{id}: Finding the episode which aired last',
+                  {'id': self.series_id})
+
+        main_db_con = db.DBConnection()
+        sql_results = main_db_con.select(
+            'SELECT '
+            '  airdate,'
+            '  season,'
+            '  episode '
+            'FROM '
+            '  tv_episodes '
+            'WHERE '
+            '  indexer = ?'
+            '  AND showid = ? '
+            '  AND airdate < ? '
+            '  AND status <> ? '
+            'ORDER BY'
+            '  airdate '
+            'DESC LIMIT 1',
+            [self.indexer, self.series_id, datetime.date.today().toordinal(), UNAIRED])
+
+        if sql_results is None or len(sql_results) == 0:
+            log.debug(u'{id}: No episode found... need to implement a show status', {'id': self.series_id})
+            self.prev_aired = u''
+        else:
+            log.debug(
+                u'{id}: Found episode {ep}', {
```

Comment:
This log seems a bit vague to me. Maybe something like `Found last aired episode` instead?

Reply:
I put it there at one time to debug. I think I can also just remove it?

Reply:
I don't mind having it, but it should be more specific imo.